src/osd/OSD.cc

```diff
@@ -6415,8 +6415,8 @@ COMMAND("bench " \
 	"name=size,type=CephInt,req=false " \
 	"name=object_size,type=CephInt,req=false " \
 	"name=object_num,type=CephInt,req=false ", \
-	"OSD benchmark: write <count> <size>-byte objects, " \
-	"(default 1G size 4MB). Results in log.",
+	"OSD benchmark: write <count> <size>-byte objects(with <obj_size> <obj_num>), " \
+	"(default count=1G default size=4MB). Results in log.",
```

Comment:
```
$ ceph tell osd.11 bench help
help not valid: help doesn't represent an int
Invalid command: unused arguments: [u'help']
bench {<int>} {<int>} {<int>} {<int>} : OSD benchmark: write <count> <size>-byte objects, (default 1G size 4MB). Results in log.
Error EINVAL: invalid command
```

On the left side it says about 4 integers, and in the description part only two of them.

Reply:
This PR includes `obj_size` and `obj_num` in the description.
```
bench {<int>} {<int>} {<int>} {<int>} :  OSD benchmark: write <count> <size>-byte objects(with <obj_size> <obj_num>), (default count=1G default size=4MB). Results in log.
```
`count` corresponds to `bytes_written` so I think that clarifies things, I'm not sure we are explicitly saying "count of block" anywhere.

Reply:
<count> bytes using <size>-byte objects

Reply:
Also, I don't understand why object size may be specified twice. Maybe they mean total size of an object and the size of a single write ?

Reply:
i agree it doesn't make sense... 4 parameters when there are really 3 underlying variables (object size, object count, and how big each write is).  but i'm not sure it's worth the effort to fix this